src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,437 @@
+| MolarActivity                 | RECOMMENDED           | [number][]    | Molar activity of compound injected. Corresponds to DICOM Tag (0018,1077) Radiopharmaceutical Specific Activity.                                                                                                                                                           |
+| MolarActivityUnit             | RECOMMENDED           | [string][]    | Unit of the specified molar radioactivity (for example, "Bq/g").                                                                                                                                                                                                           |
+| MolarActivityMeasTime         | RECOMMENDED           | [string][]    | Time to which molar radioactivity measurement above applies in the default unit "hh:mm:ss".                                                                                                                                                                                |
+| InfusionRadioactivity                 | RECOMMENDED           | [number][]    | Amount of radioactivity infused into the patient. This value must be less than or equal to the total injected radioactivity (`InjectedRadioactivity`). Units should be the same as `InjectedRadioactivityUnit`.                                                                                                                                                                                                                                             |
+| InfusionStart                 | RECOMMENDED           | [number][]    | Time of start of infusion with respect to `TimeZero` in the default unit seconds.                                                                                                                                                                                                                                             |
+| InfusionSpeed                 | RECOMMENDED           | [number][]    | If given, infusion speed.                                                                                                                                                                                                                                                  |
+| InfusionSpeedUnit             | RECOMMENDED           | [string][]    | Unit of infusion speed (for example, "mL/s").                                                                                                                                                                                                                              |
+| InjectedVolume            | RECOMMENDED           | [string][]    | Injected volume of the radiotracer in the unit ´mL´.                                                                                                                                                                                                        |
```

Comment:
code formatting must be done in backticks: ` not with "forwardticks" like here: ´

`like so`

´not like this´

```suggestion
| InjectedVolume            | RECOMMENDED           | [string][]    | Injected volume of the radiotracer in the unit `mL`.                                                                                                                                                                                                        |
```

Reply:
I was just going through it and changing - you are simply too fast and efficient, @sappelhoff ! :-)